webui/react/src/hooks/usePermissions.ts

```diff
@@ -261,6 +263,14 @@ const canViewModelRegistry = ({
   return !rbacEnabled || rbacReadPermission || permitted.has(V1PermissionType.VIEWMODELREGISTRY);
 };
 
+const canViewModelWorkspace = (
```

Comment:
minor nit: `canCreateModelWorkspace` is a bit more clear concerning what this permission is for.